src/draive/conversation/completion/__init__.py

```
@@ -0,0 +1,3 @@
+from draive.conversation.completion.state import Conversation
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)